docs/references/react/use-organization-list.mdx

```diff
@@ -1,3 +1,267 @@
 ---
-sanity_slug: /docs/reference/clerk-react/useorganizationlist
+title: useOrganizationList()
+description: Access methods for switching between active organizations and for listing available memberships, invitations and suggestions.
 ---
+
+import { Callout } from "nextra-theme-docs";
+import { Tables } from "@/components/Table";
+import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+
+# `useOrganizationList()`
+
+Access methods for switching between active organizations and for listing available memberships, invitations and suggestions.
+
+`useOrganizationList` enables developers to fetch the memberships, invitations or suggestsions for an active user.
+
+`Note: these resources are paginated`
+
+The hook requires developers to opt-in by specifying which resource they need to fetch and paginate through. It is ideal for infinite lists or tables
+
+## Usage
+### Infinite list of joined organizations
+<CodeBlockTabs options={['Next.js', 'React']}>
+```tsx copy filename="pages/joined-organizations.tsx"
+import { useOrganizationList } from "@clerk/nextjs";
+import React from "react";
+
+const JoinedOrganizationList = () => {
+  const { isLoaded, setActive, userMemberships } = useOrganizationList({
+    userMemberships: {
+      infinite: true,
+    },
+  });
+
+  if (!isLoaded) {
+    return <>Loading</>;
+  }
+
+  return (
+    <>
+      <ul>
+        {userMemberships.data?.map((mem) => (
+          <li key={mem.id}>
+            <span>{mem.organization.name}</span>
+            <button
+              onClick={() => setActive({ organization: mem.organization.id })}
+            >
+              Select
+            </button>
+          </li>
+        ))}
+      </ul>
+
+      <button
+        disabled={!userMemberships.hasNextPage}
+        onClick={() => userMemberships.fetchNext()}
+      >
+        Load more
+      </button>
+    </>
+  );
+};
+
+export default JoinedOrganizationList;
+```
+
+```tsx copy filename="joined-organizations.tsx"
+import { useOrganizationList } from "@clerk/nextjs";
+import React from "react";
+
+const JoinedOrganizationList = () => {
+  const { isLoaded, setActive, userMemberships } = useOrganizationList({
+    userMemberships: {
+      infinite: true,
+    },
+  });
+
+  if (!isLoaded) {
+    return <>Loading</>;
+  }
+
+  return (
+    <>
+      <ul>
+        {userMemberships.data?.map((mem) => (
+          <li key={mem.id}>
+            <span>{mem.organization.name}</span>
+            <button
+              onClick={() => setActive({ organization: mem.organization.id })}
+            >
+              Select
+            </button>
+          </li>
+        ))}
+      </ul>
+
+      <button
+        disabled={!userMemberships.hasNextPage}
+        onClick={() => userMemberships.fetchNext()}
+      >
+        Load more
+      </button>
+    </>
+  );
+};
+
+export default JoinedOrganizationList;
+```
+</CodeBlockTabs>
+
+### Table of invitations
+<CodeBlockTabs options={['Next.js', 'React']}>
```

Comment:
We should have app router and pages router examples for Next.js,
so we can use for Next.js versus React examples,
and for the Next.js tab, add <CodeBlockTabs options={["App router", "Pages router"}]